packages/core/admin/package.json

```diff
@@ -46,7 +46,6 @@
     "@fingerprintjs/fingerprintjs": "3.3.3",
     "@fortawesome/fontawesome-free": "^5.15.3",
     "@fortawesome/fontawesome-svg-core": "^1.2.35",
-    "@fortawesome/free-brands-svg-icons": "^5.15.3",
     "@fortawesome/free-solid-svg-icons": "^5.15.3",
     "@fortawesome/react-fontawesome": "^0.2.0",
     "@pmmmwh/react-refresh-webpack-plugin": "0.5.7",
```

Comment:
This one needs an update to `2.0.0`, since jest was upgraded to v28 in https://github.com/strapi/strapi/pull/13740